packages/http-specs/specs/stream/jsonl/main.tsp

```diff
@@ -0,0 +1,29 @@
+import "@typespec/http";
+import "@typespec/http/streams";
+import "@typespec/spector";
+
+using Http;
+using Http.Streams;
+using Spector;
+
+@doc("Test of jsonl stream.")
+@scenarioService("/stream/jsonl")
+namespace Stream.Jsonl;
```

Comment:
> when regen doc for spector, i encountered this error. @timotheeguerin is it a problem in spector or in http lib?
![image](https://github.com/user-attachments/assets/c549b953-1ed6-47ba-9e5f-7c58978ca331)

replying here so we can have a thread. I think this is the same problem the Azure specs have where the namespace conflict with an existing namespace from a library and we add to add `_Specs_` prefix